### PR TITLE
Enable RuboCop for template files

### PR DESCRIPTION
**Pull request by revolter**
### Checklist
- [ ] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### CODE REVIEW

File: precheck/lib/assets/PrecheckfileTemplate [hunk 1]
```diff
@@ -24,4 +24,4 @@
 # placeholder_text(level: :error)
 # test_words(level: :error)
 # unreachable_urls(level: :error)
-# custom_text(data: ["fabric"], level: :warn)
+# custom_text(data: ['fabric'], level: :warn)
```
Comment (KrauseFx) [inline precheck/lib/assets/PrecheckfileTemplate]: why that change?

File: fastlane/lib/assets/DefaultFastfileTemplate [hunk 2]
```diff
@@ -11,6 +11,6 @@
 
 default_platform(:ios)
 
-platform :ios do
+platform(:ios) do
```
Comment (KrauseFx) [inline fastlane/lib/assets/DefaultFastfileTemplate]: I do have to say, that we should make an exception for `platform` and `lane`, as it looks much nicer without brackets, and we don't need to let people know it's actually a method call

File: snapshot/lib/assets/SnapfileTemplate [hunk 3]
```diff
@@ -2,32 +2,32 @@
...
 # ])
 
 # languages([
-#   "en-US",
-#   "de-DE",
-#   "it-IT",
-#   ["pt", "pt_BR"] # Portuguese with Brazilian locale
+#   'en-US',
+#   'de-DE',
+#   'it-IT',
```
Comment (KrauseFx) [inline snapshot/lib/assets/SnapfileTemplate]: Why the quote changes?

File: precheck/lib/assets/PrecheckfileTemplate (hunk 1 above)
Comment (revolter) [reply precheck/lib/assets/PrecheckfileTemplate]: Sorry, I thought `Style/StringLiterals` is enabled. I'll revert them.

File: fastlane/lib/assets/DefaultFastfileTemplate (hunk 2 above)
Comment (revolter) [reply fastlane/lib/assets/DefaultFastfileTemplate]: Sure

**Comment by revolter** [comment]
@KrauseFx, Fixed. If the rest of the changes don't make sense, feel free to close this.

**Review by minuscorp — APPROVED** [review]
It looks great to me!! 🚀

**Pull request merged**